pubspec.yaml

```diff
@@ -16,8 +16,8 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  http: ^0.13.0
```

Comment:
Would be better to leave the upper bound at 1.1.0 instead. It is the current stable release and 1.2 could have breaking changes that will cause issues if the SuperTokens SDK is not compatible with it when it comes out

Reply:
The upperbound is set to 1.1.X; the upperbound is now specifically set to not install 1.2.0 if they release that

Reply:
Also @nkshah2 if they did release a breaking change, wouldn't that go to 2.0 instead of 1.2 (for the http lib)?

Reply:
@DwayneCoussement Youre right, I think we should still keep the lower bound to 0.13 to not force users to have to update